### ISSUE TITLE
Typography: Font size

### ISSUE THREAD

**Issue by imphil** [fossi-foundation/fossi-foundation-web#5]
Two questions on font sizes:

* The web design uses a 20px font for "large links" (without scaling for phone views), which does not appear in the style/typography page (https://zeroheight.com/822235964/p/852315-typography). Should we add 20px there as intermediate variant?
* Heading font sizes are scaled to their 1.25x variant for the phone view, but the large paragraph (for example) is not scaled. Should we use the same scaling for phone views for all components where we use font sizes > 16px to keep the aspect ratio in place? 
 A good example that motivates the consistent scaling is https://fossi-foundation-preview.netlify.app/about-us: the heading ("[Who we are](https://fossi-foundation-preview.netlify.app/about-us#who-we-are)") is scaled down from 36px to 25px, but the large paragraph below it would be kept at 24px -- which is fine on desktop, but not on mobile. I currently implement responsive font scaling for the large paragraph as well, so in the preview at [Who we are](https://fossi-foundation-preview.netlify.app/about-us#who-we-are) the large paragraph font is scaled down to 20px.

**Comment by asherdub** [comment]
@imphil  the 1.5x type scale is for general use across print and screen. Web type has more options to accommodate responsive design etc. I have added a note explaining this https://zeroheight.com/822235964/p/852315-typography/t/427f7b which links to the tokens page.

I have added another style to https://zeroheight.com/822235964/p/48281f-typography/t/729e31 for large paragraphs on phone that uses 20px as suggested. Links should not reduce in size for phone though.
